microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -32,46 +32,89 @@ See also:
 
 
 <!-- ====================================================================== -->
-## Get started
+## Open the Performance tool
 
-In the following tutorial, you open DevTools on a "Sluggish Animation" demo page and use the **Performance** tool to find a performance bottleneck on the page.
+In the sections below, you open DevTools on the "Sluggish Animation" demo page and use the **Performance** tool to find a performance bottleneck on the page.
 
-1. Open the [Sluggish Animation](https://microsoftedge.github.io/Demos/devtools-performance-get-started/) demo page in your InPrivate tab or window.  To do that, right-click the link and then select **Open link in InPrivate window**.  You'll profile this page, which shows a variable number of icons moving up and down. For more information about InPrivate, see [Browse InPrivate in Microsoft Edge](https://support.microsoft.com/microsoft-edge/browse-inprivate-in-microsoft-edge-cd2c9a48-0bc4-b98e-5e46-ac40c84e27e2)
+1. Right-click the link [Sluggish Animation](https://microsoftedge.github.io/Demos/devtools-performance-get-started/) and then select **Open link in InPrivate window**.  You'll profile this page, which shows a variable number of icons moving up and down.  See also [Browse InPrivate in Microsoft Edge](https://support.microsoft.com/microsoft-edge/browse-inprivate-in-microsoft-edge-cd2c9a48-0bc4-b98e-5e46-ac40c84e27e2).
 
-   Source code: [MicrosoftEdge / Demos > devtools-performance-get-started](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-performance-get-started).
+   If you want to view the source code for the demo, see [MicrosoftEdge / Demos > devtools-performance-get-started](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-performance-get-started).
 
+1. Right-click the demo webpage and then select **Inspect**.
 
-   <!--TODO: replace section when updated for Chromium-based Edge  -->
+   DevTools opens:
 
-   <!-- You can view the source files for the "Sluggish Animation" demo page at the [MicrosoftEdge/Demos > devtools-performance-get-started](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-performance-get-started) repo folder. -->
+   ![The demo on the left, and DevTools on the right](./index-images/get-started-side-by-side.png)
 
-1. Press **Ctrl+Shift+I** (Windows, Linux) or **Command+Option+I** (macOS) to open DevTools:
+Most screenshots of the **Performance** tool show DevTools [undocked to a separate window](../customize/placement.md).
 
-   ![The demo on the left, and DevTools on the right](./index-images/get-started-side-by-side.png)
 
-For the rest of the screenshots, DevTools is shown [undocked to a separate window](../customize/placement.md).
+<!-- ====================================================================== -->
+## View local and field metrics
+
+todo: provisionally added new section from https://developer.chrome.com/blog/new-in-devtools-130#live-metrics-recommendations
```

Comment:
in this old tut article, maybe revert this added h2 section.  compare & sync this article with recent v129 Chrome tut:
**Analyze runtime performance**
https://developer.chrome.com/docs/devtools/performance